core/src/main/resources/lib/form/enum.jelly

```diff
@@ -36,9 +36,14 @@ THE SOFTWARE.
       The name of the enum to set as default value for the first configuration.
     </st:attribute>
   </st:documentation>
-  <select class="setting-input" name="${field}">
-    <j:forEach var="it" items="${descriptor.getPropertyType(instance,field).enumConstants}">
-      <f:option value="${it.name()}" selected="${instance != null ? it==instance[field] : it.name()==default}">
+
+  <j:if test="${attrs.field==null}">
+    <j:set target="${attrs}" property="field" value="${entry.field}" />
```

Comment:
`${entry.field}` always puts null here. As designed, right?

Reply:
This fragment is also in `core/src/main/resources/lib/form/descriptorList.jelly`.

It looks up the ancestor `<entry>` set by `entry.jelly`.
`attrs.field` is not the same as `entry.field` which most probably will not be null.
If enum field and parent entry field are both null then the form is just incorrectly written.

Please look at the following examples
```
  <f:entry field="updateStrategy">
    <f:enum>${it.label}}</f:enum>
  </f:entry>
```
```
  <f:enum field="updateStrategy">${it.label}}</f:enum>
```

Reply:
NVM. I misread the code. Thanks for clarification!